tensorflow/python/ops/variable_scope_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
(that includes using indentation and whitespace consistent with how we do it everywhere else)